esp/esp/program/models/__init__.py

```diff
@@ -281,6 +281,7 @@ class Program(models.Model, CustomFormsLinkModel):
     class Meta:
         app_label = 'program'
         db_table = 'program_program'
+        ordering = ['-id','name']
```

Comment:
Right, I support sorting "chronologically".  But there are no ties to break on `id`, so unless I'm misunderstanding something, adding a second sort cannot possibly change the sort order.